### DISCUSSION TITLE
Show Downloads

### DISCUSSION THREAD

**Discussion by bagnos** [MicrosoftEdge/WebView2Feedback#3938] [Q&A]
Hi,

I have an accessibility issue about Download Section in Webview2.
In Microsoft Edge browser that section is accessible via the combination of the CTRL + J keys.
I noticed that this combination does not work in webview2, are there any ways to make the download section accessible by visually impaired people?

Simone

**Comment by jarno9981** [comment]
Yes thats possible i will send to code in a minutes

**Comment by jarno9981** [comment]
@bagnos 

here

//open dialog
private void ToggleDefaultDownloadDialog()
        {
            if (newWeb.CoreWebView2.IsDefaultDownloadDialogOpen == true)
            {
                newWeb.CoreWebView2.CloseDefaultDownloadDialog();
            }
            else
            {
                newWeb.CoreWebView2.OpenDefaultDownloadDialog();
                SetDefaultDownloadDialogPosition();
            }
        }

anchor dialog
        private void SetDefaultDownloadDialogPosition()
        {
            const int defaultMargin = 20;
            CoreWebView2DefaultDownloadDialogCornerAlignment cornerAlignment
                = CoreWebView2DefaultDownloadDialogCornerAlignment.TopRight;
            System.Drawing.Point margin = new System.Drawing.Point(
                defaultMargin, defaultMargin);
            newWeb.CoreWebView2.DefaultDownloadDialogCornerAlignment =
                cornerAlignment;
            newWeb.CoreWebView2.DefaultDownloadDialogMargin = margin;
        }

**Comment by bagnos** [comment]
Thanks @jarno9981 

How can that code help me about the accessibility issue of Download Section?

I need to select each downloaded file in Download Section by pressing a key combination (es. CTRL + J  like Microsoft Edge).

**Comment by jarno9981** [comment]
@bagnos this code is Needed for keys
Web is the name of your webview2 

Web.corewebview2.settings.AreBrowserAcceleratorKeysEnabled = true if you want to use key combinations

This will enable all the build in shortcut keys

You have to put in initilizationcompleted

**Comment by bagnos** [comment]
Thanks @jarno9981 but that solution doesn't work.
CTRL + J is not a key combination  enabled in WebView2, I think also that the property AreBrowserAcceleratorKeysEnabled is enabled by deafult.

**Comment by jarno9981** [comment]
> Thanks @jarno9981 but that solution doesn't work.
> CTRL + J is not a key combination  enabled in WebView2, I think also that the property AreBrowserAcceleratorKeysEnabled is enabled by deafult.

@bagnos okay thats what i now from the sheets specs end if you make a virtual key press event and use ctrl + j maybe that works

And then use the opendefaultdialog log that i mention
